classes/mender-uboot.bbclass

```diff
@@ -7,6 +7,3 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 IMAGE_BOOT_FILES_append = " ${IMAGE_BOOT_ENV_FILE}"
 
 EXTRA_IMAGEDEPENDS += "u-boot"
-
-DEPENDS_mender_append = " u-boot u-boot-fw-utils"
-RDEPENDS_mender_append = " u-boot u-boot-fw-utils"
```

Comment:
In its out-of-the-box shape, yes. But there are several reasons for it:

1. Later we will provide ways for people to call user defined scripts as commands for switching partition, updating boot counter, etc.
2. People are already testing Mender without U-Boot. Definitely an experimental thing to do at this point, but should still be possible.